spec/02_interaction_spec.rb

```diff
@@ -8,12 +8,12 @@
     fill_in("pirate[name]", :with => contents[0])
     fill_in("pirate[weight]", :with => contents[1])
     fill_in("pirate[height]", :with => contents[2])
-    fill_in("pirate[ships][0][name]", :with => contents[3])
-    fill_in("pirate[ships][0][type]", :with => contents[4])
-    fill_in("pirate[ships][0][booty]", :with => contents[5])
-    fill_in("pirate[ships][1][name]", :with => contents[6])
-    fill_in("pirate[ships][1][type]", :with => contents[7])
-    fill_in("pirate[ships][1][booty]", :with => contents[8])
+    fill_in("ship_name_1", :with => contents[3])
```

Comment:
This contents array is super weird. It should be a hash with key/values so like `contents["name"]="Joe"`